src/index.ts

```diff
@@ -218,3 +214,4 @@ export {
 export { DetailPageCard, DetailPageCardProps, Content, ContentType } from './components/DetailPageCard/DetailPageCard'
 export { SimpleLogViewer, SimpleLogViewerProps } from './components/LogViewer/SimpleLogViewer'
 export { breakOnLinks, LogLineProps, LogLine, MemoizedLogLine } from './components/LogViewer/LogLine'
+export * from '@harness/design-system'
```

Comment:
Adding this for now because we will have to change many imports in NextGenUI. Almost every file we are using Color, we can take this change in the Next Pr, once these changes are tested
@abhinavrastogi-harness @vkbansal